cypress/support/e2e.ts

```diff
@@ -89,28 +89,32 @@ Cypress.Commands.overwrite(
 )
 
 beforeEach(() => {
-  // Infura security policies are based on Origin headers.
-  // These are stripped by cypress because chromeWebSecurity === false; this adds them back in.
-  cy.intercept(/infura.io/, (res) => {
-    res.headers['origin'] = 'http://localhost:3000'
-    res.alias = res.body.method
-    res.continue()
+  // many of our api calls enforce these headers come from our app
+  cy.intercept('*', (req) => {
+    req.headers['referer'] = 'https://app.uniswap.org'
+    req.headers['origin'] = 'https://app.uniswap.org'
   })
+    // Infura security policies are based on Origin headers.
+    // These are stripped by cypress because chromeWebSecurity === false; this adds them back in.
+    .intercept(/infura.io/, (req) => {
+      req.headers['referer'] = 'http://localhost:3000'
+      req.headers['origin'] = 'http://localhost:3000'
+      req.alias = req.body.method
+      req.continue()
+    })
 
-  // Graphql security policies are based on Origin headers.
-  // These are stripped by cypress because chromeWebSecurity === false; this adds them back in.
-  cy.intercept('https://api.uniswap.org/v1/graphql', (res) => {
-    res.headers['origin'] = 'https://app.uniswap.org'
-    res.continue()
-  })
-  cy.intercept('https://beta.api.uniswap.org/v1/graphql', (res) => {
-    res.headers['origin'] = 'https://app.uniswap.org'
-    res.continue()
-  })
-
-  cy.intercept('https://api.uniswap.org/v1/amplitude-proxy', (res) => {
-    res.reply(JSON.stringify({}))
-  })
+    .intercept('https://api.uniswap.org/v1/amplitude-proxy', (req) => {
```

Comment:
```suggestion
    // Mock Amplitude responses to avoid analytics from tests. 
    .intercept('https://api.uniswap.org/v1/amplitude-proxy', (req) => {
```